test/commandLine/commandLineTest.js

```diff
@@ -0,0 +1,519 @@
+/* eslint-env mocha */
+
+const assert = require('assert')
+const fs = require('fs')
+const fse = require('fs-extra')
+const path = require('path')
+const rimraf = require('rimraf')
+const fork = require('child_process').fork
+
+describe('Command Line Tests', function () {
+  const appDir = path.join(__dirname, '../app/commandLineTest')
+
+  before(function (done) {
+    fse.ensureDirSync(path.join(appDir))
+
+    fs.readFile(path.join(__dirname, '/app.txt'), (err, buffer) => {
+      if (err) {
+        throw err
+      }
+      fs.writeFile(path.join(appDir, '/app.js'), buffer.toString(), (error) => {
+        if (error) {
+          throw error
+        }
+      })
+    })
+    done()
+  })
+
+  after(function (done) {
+    rimraf(appDir, (err) => {
+      if (err) {
+        throw err
+      } else {
+        done()
+      }
+    })
+  })
+
+  /* checking soon to be deprecated flags */
```

Comment:
These commented category names could be individual describe blocks, that way we can see them in the mocha console output.